runtime/kusama/src/lib.rs

```diff
@@ -1397,6 +1398,63 @@ impl pallet_gilt::Config for Runtime {
 	type WeightInfo = weights::pallet_gilt::WeightInfo<Runtime>;
 }
 
+pub struct BalanceToU256;
+impl sp_runtime::traits::Convert<Balance, sp_core::U256> for BalanceToU256 {
+	fn convert(n: Balance) -> sp_core::U256 {
+		n.into()
+	}
+}
+pub struct U256ToBalance;
+impl sp_runtime::traits::Convert<sp_core::U256, Balance> for U256ToBalance {
+	fn convert(n: sp_core::U256) -> Balance {
+		n.try_into().unwrap_or(Balance::MAX)
```

Comment:
Is this a defensive?

Reply:
Both yes and no: OTTMY this will when the total issuance of Polkadot grows equivalent of like 100 years or so. 

For now, I am going to mark it as defensive anyways, that's a good practice. 

@emostov this something that you can chime in if you still recall the details.